app/parse.py

```diff
@@ -8,8 +12,46 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_html: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_html.select_one(".text").text,
+        author=quote_html.select_one(".author").text,
+        tags=[tag.text for tag in quote_html.select(".tag")],
+    )
+
+
+def parse_all_quotes_on_page(url: str) -> list:
```

Comment:
list of what?
Also as for me better to do this as class method
Also parse_single_quote better to do classmethod of Quote